draft-ietf-tls-wkech.xml

```diff
@@ -160,24 +161,56 @@
 
     <section title="Example use of the well-known URI for ECH">
 
-        <t>An example deployment could be as follows:
+        <t>An example deployment could be as follows:</t>
+
+<figure title="Shared-Mode Topology with Zone Factory and DNS" anchor="shared-mode"><artwork><![CDATA[
+                    +--------------------+
+                    |                    |
+                    |   2001:DB8::1111   |
+Client <----------->|                    |
+                    |   cfs.example.com  |
+                    | backend.example.com|
+                    +--------------------+
+                     Client-Facing Server 
+                        |            |
+     (1) ZF reads       |            |  (2) ZF checks
+         .well-known    V            V      ECHConfig
+                    +--------------------+
+                    |                    |
+                    |   zf.example.com   |
```

Comment:
This might help to hint that the zone factory is affiliated with the nameserver, not the backend or CFS.

```suggestion
                    |   zf.example.net   |
```